### PR TITLE
Np 45136 cristin import map notes fields

### PR DESCRIPTION
**Pull request by anetteOlli**

### CODE REVIEW

File: buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle [hunk 1]
```diff
@@ -34,6 +34,7 @@ java {
 
 repositories {
     mavenCentral()
+    mavenLocal()
```
Comment (brinxmat) [inline buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle:37]: Preferably not

File: buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle [hunk 2]
```diff
@@ -43,7 +44,7 @@ tasks.named('test') {
     useJUnitPlatform {
         excludeTags "RemoteTest", "integrationTest", "KarateTest"
     }
-    failFast = true
+    failFast = false
```
Comment (brinxmat) [inline buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle:47]: The false setting likely gives us a clearer picture in CI reports

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle (hunk 2 above)
Comment (anetteOlli) [reply buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle]: Fjernet

File: buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle (hunk 1 above)
Comment (anetteOlli) [reply buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle]: fjernet

**Review by brinxmat — APPROVED** [review]

**Pull request merged**